crates/language/Cargo.toml

```diff
@@ -30,6 +30,7 @@ async-trait.workspace = true
 async-watch.workspace = true
 clock.workspace = true
 collections.workspace = true
+ec4rs = "1.1"
```

Comment:
Let’s make this a workspace dependency.

Reply:
The root Cargo.toml in Zed usually has all the version numbers specified, and then other Cargo.toml declare dependencies as `dependency.workspace = true`